iina/CropSettingsViewController.swift

```diff
@@ -71,6 +71,14 @@ class CropSettingsViewController: NSViewController {
       playerCore.setCrop(fromFilter: filter)
       // custom crop has no corresponding menu entry
       playerCore.info.unsureCrop = ""
+      self.mainWindow.displayOSD(.crop("\(self.cropx),\(self.actualCropy) \(self.cropw)x\(self.croph)"))
+    }
+  }
+
+  @IBAction func cancelBtnAction(_ sender: AnyObject)
+  {
+    mainWindow.exitInteractiveMode{
+      return
```

Comment:
So sorry. It will be better if code style is in the  CONTRIBUTING.md. If so others won't make the same mistake as I do